src/formatter/helpers/issue_helpers.js

```diff
@@ -93,6 +93,18 @@ function formatStep({
   }
   text += '\n'
 
+  if (Array.isArray(testStep.attachments)) {
+    testStep.attachments
+      .filter(({ media }) => media.type === 'text/plain')
+      .forEach(({ data }) => {
+        text +=
+          indentString(
+            colorFns[Status.UNDEFINED](figures.info + ' ' + data),
```

Comment:
I'm uncertain about attachments always being yellow. Thoughts on it being the same color as the step or not having any color?

Reply:
Ok for no color.